mailserver/mailserver.go

```diff
@@ -200,5 +237,57 @@ func (s *WMailServer) validateRequest(peerID []byte, request *whisper.Envelope)
 
 	lower := binary.BigEndian.Uint32(decrypted.Payload[:4])
 	upper := binary.BigEndian.Uint32(decrypted.Payload[4:8])
+
+	lowerTime := time.Unix(int64(lower), 0)
+	upperTime := time.Unix(int64(upper), 0)
+	if upperTime.Sub(lowerTime) >= maxQueryRange {
```

Comment:
`>` as equal 24 hours is valid. We are using exactly 24 hours in the client now.